src/DropDown/DropDown.stories.tsx

```diff
@@ -29,6 +29,21 @@ const listItemArray = [
     { key: 5, label: 'List Item 5' },
 ];
 
+const positions: Position[] = [
```

Comment:
It's annoying you have to do this (as in, I feel bad that you have to do this). There is an export of of `type Position` in `src/utils/position/`. Maybe it's worth moving this in there and exporting it so that we keep all these values together. Possibly make it an enum and convert it to `type`, ala `export type Position = keyof typeof positions;`?

Reply:
Gonna forgo this for now.